config/profile.go

```diff
@@ -651,15 +652,19 @@ func (p *Profile) SetRootPath(rootPath string) {
 func (p *Profile) resolveSourcePath(sourceBase string, sourcePaths ...string) []string {
 	var applySourceBase, applyBaseDir pathFix
 
-	// Backup source is NOT relative to the configuration, but to PWD or sourceBase (if not empty)
-	// Applying "sourceBase" if set
-	if sourceBase = strings.TrimSpace(sourceBase); sourceBase != "" {
```

Comment:
I see an issue with this change. Originally all actions happened when trimmed sourceBase base not empty. With the new logic, sourceBase is formatted and may have a value when it was set to a space char.

To avoid side effects, I'd prefer to leave the original logic and instead set `applySourceBase` and `applyBase` to `nil` when the new option is set.

Reply:
Ok.. need to admit looking at the entire method this side effect does not matter as only the `apply*` are used. In that case just wrap the entire block into the if condition (without modification).